apps/nextra/theme.config.tsx

```diff
@@ -58,17 +58,13 @@ const config: DocsThemeConfig = {
     content: function useBanner() {
       return (
         <div>
-          🎉 Welcome! Translations are currently experimental. |
-          翻訳は現在実験的です。 | 翻译目前处于实验阶段。
-          <br />
+          🎉 Welcome to the new Aptos Docs!{" "}
           <a
             className="text-blue-400 dark:text-blue-400 hover:underline"
             target="_blank"
-            href="https://github.com/aptos-labs/developer-docs/issues/new/choose"
+            href="https://forms.gle/h7CGG9UKA1zCRNgd6"
```

Comment:
I don't think anyone is looking at the google form.  Let's keep it as a GitHub issue

Reply:
I added the suggestions for ease below